### DISCUSSION TITLE
eRegulon threshold

### DISCUSSION THREAD

**Discussion by Abigail575** [aertslab/scenicplus#45] [Q&A] [answered]
Hello, thanks for the scenicplus package :) I was wondering if there is a way of obtaining a regulon for a TF of interest that is not shown in the eRegulon metadata as it presumably gets pruned at some point? Is there a way of playing with this threshold in order to still obtain a list of target genes for a TF that may not itself be enriched in the cell population (as it has an AUCell score below the threshold) but its targets can still be inferred? Thanks!

**Comment by SeppeDeWinter** [comment, marked as answer]
Hi @Abigail575 

Most importantly to find a TF of interest is to make sure that you have a motif for the given TF. 
SCENIC+ is only able to find eRegulons for TFs for which the motif is enriched.

So I would suggest first checking your motif enrichment results to see wether you can find your TF of interest there. If you can not then there is no chance of finding an eRegulon for that TF. In that case try playing with the region sets used for motif enrichment (try a different set of DARs, different thresholding method for the topics, ...).

If you can find an enriched motif for your TF of interest then it is probably filtered out because it has very few target genes. By default we filter out eRegulons which have less than 10 target genes. You can change this parameter in the `build_grn` by setting the `min_target_genes` parameter.

I hope this helps.

Best,

Seppe

**Reply by Abigail575** [reply]
Thank you so much!
Abigail
